src/facets/DiamondManagerFacet.sol

```diff
@@ -86,6 +90,13 @@ contract DiamondManagerFacet {
         if (receivers.length != proportion.length) {
             revert DiamondManagerFacet__Invalid_Input();
         }
+        uint256 totalProportion = 0;
+        for (uint256 i = 0; i < proportion.length; i++) {
```

Comment:
no need to initialise i to 0, also to keep it consistent I'd rename proportions to shares